README.md

```diff
@@ -99,7 +105,7 @@ try {
 
 ## 3. Email Suggestion Aggregation
 
-On page load and detecting the RP has performed (2), for each registered Issuer:
+On page load and detecting the RP has performed (2), for each registered Issuer that the user is still [logged-in](https://w3c-fedid.github.io/login-status/#get-the-login-status) to:
 
```

Comment:
this indicates that I won't be able to select a verified email if I happen to be logged out :(